specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,275 @@ with interoperability issues.
+model defines a three-dimensional coordinate system with weight, stretch, and style as the axes. Each
+font in a WWS family must have a unique location defined by its coordinates along those three axes. To
+select a font, one specifies a WWS family name and weight, stretch, and style parameters.
+
+By contrast, the typographic font family model has an N-dimensional design space. A font designer can 
+define any number of design axes, each identified by a four-character _axis tag_. A given font’s location 
+in the N-dimensional design space is defined by an array of _axis values_, where each axis value comprises 
+an axis tag and a floating-point value. To select a font, one specifies a typographic family name and an 
+array of axis values (`DWRITE_FONT_AXIS_VALUE` structures).
+
+Although the number of font axes is open-ended, there are a few registered axes with standard meanings,
+and the weight, stretch, and style values can be mapped to registered axis values. `DWRITE_FONT_WEIGHT`
+can be mapped to a "wght" (`DWRITE_FONT_AXIS_TAG_WEIGHT`) axis value. `DWRITE_FONT_STRETCH` can be mapped
+to a "wdth" (`DWRITE_FONT_AXIS_TAG_WIDTH`) axis value. `DWRITE_FONT_STYLE` can be mapped to a combination
+of "ital" and "slnt" (`DWRITE_FONT_AXIS_TAG_ITALIC` and `DWRITE_FONT_AXIS_TAG_SLANT`) axis values. 
+
+Another registered axis is "opsz" (`DWRITE_FONT_AXIS_TAG_OPTICAL_SIZE`). An optical font family such as
+Sitka includes fonts that differ along the "opsz" axis, meaning they are designed to be used at different
+point sizes. The WWS font family model does not have an optical size axis, so the Sitka font family must
+be split into multiple WWS font families: "Sitka Small", "Sitka Text", "Sitka Subheading", and so on. Each
+WWS font family corresponds to a different optical size, and it's left to the user to specify the right
+WWS family name for a given font size. With the typographic font family model, the user can simply choose
+"Sitka", and the application can automatically set the "opsz" axis value based on the font size.
+
+Advantages of the typographic font family model are:
+
+  - Fonts can be grouped into families as intended by the designer, rather than being split into
+    subfamilies due to limitations of the font family model.
+
+  - An application can automatically select the correct "opsz" axis value based on the font size
+    rather than exposing different optical sizes to the user as different font families.
+
+  - Arbitrary instances of variable fonts can be selected. For example, if a variable font supports
+    weights in the continuous range 100-900, the typographic model can select _any_ weight in this range.
+    The older font family models can only choose the nearest weight from among the named instances defined
+    by the font.
+
+However, migrating to the typographic font family model raises potential compatibility issues. The
```

Comment:
From these paragraphs it's not 100% clear to me how the typographic family model captures both static instances with certain "classic" attributes and variable fonts. Perhaps it'd make sense to add some clarification on that? (I may have missed something).

Reply:
For variable fonts, an `IDWriteFontFace` object always represents a particular instance of the variable font. It may be a named instance or an arbitrary instance, but the axis values are specified at the time the font face object is created. That way, the `IDWriteFontFace` interface can work exactly the same for variable fonts as for static fonts when it comes to getting metrics, rendering glyphs, etc. Font matching returns a prioritized list of `IDWriteFontFace` objects based on the input family name and axis values. Thus, the mapping of input axis values to variable font instances happens at font matching time.